src/main/java/serverutils/ServerUtilitiesConfig.java

```diff
@@ -544,6 +544,14 @@ public boolean log(EntityPlayerMP player) {
         @Config.DefaultBoolean(false)
         public boolean show_playtime;
 
+        @Config.Comment("Enabled Player Sleeping Percentage to skip night. Use the gamerule playersSleepingPercentage to set the percentage.")
+        @Config.DefaultBoolean(false)
+        public boolean enable_player_sleeping_percentage;
+
+        @Config.Comment("Default Player Sleeping Percentage")
+        @Config.DefaultString("50")
+        public String default_player_sleeping_percentage;
```

Comment:
Why is this config a string? An int or a double would make more sense since you can also give it a range between 0-100.
Is the "default" in the name and comment necessary?

Reply:
Ah I get the reason for the default since you would have to change the gamerule and not this setting.
Is this config even necessary then?

Reply:
I made it a string because the game rule only takes strings, but I could make the config an int and convert that to a string for the gamerule.